Dockerfile

```diff
@@ -1,76 +1,15 @@
-FROM ubuntu:14.04
+FROM cs50/common
 
-# packages
-RUN apt-get update && \
-    apt-get install -y software-properties-common && \
-    add-apt-repository -y ppa:git-core/ppa && \
-    apt-get update && \
-    apt-get install -y \
-        apt-transport-https \
-        clang \
-        curl \
-        git \
-        sqlite3 \
-        unzip \
-        valgrind \
-        wget
-
-# install Python 3.6
-# https://github.com/yyuu/pyenv/blob/master/README.md#installation
-# https://github.com/yyuu/pyenv/wiki/Common-build-problems
-ENV PYENV_ROOT /opt/pyenv
-RUN apt-get install -y \
-        build-essential \
-        curl \
-        libbz2-dev \
-        libncurses5-dev \
-        libncursesw5-dev \
-        libreadline-dev \
-        libsqlite3-dev \
-        libssl-dev \
-        llvm \
-        wget \
-        xz-utils \
-        zlib1g-dev && \
-    wget -P /tmp https://github.com/yyuu/pyenv/archive/master.zip && \
-    unzip -d /tmp /tmp/master.zip && \
-    rm -f /tmp/master.zip && \
-    mv /tmp/pyenv-master /opt/pyenv && \
-    chmod a+x /opt/pyenv/bin/pyenv && \ 
-    /opt/pyenv/bin/pyenv install 3.6.0 && \
-    /opt/pyenv/bin/pyenv rehash && \
-    /opt/pyenv/bin/pyenv global 3.6.0
-ENV PATH "$PYENV_ROOT"/shims:"$PYENV_ROOT"/bin:"$PATH"
-
-# install CS50-specific packages
-RUN pip install cs50 check50 style50 submit50
+# nltk and punkt data
+RUN nltk && python -m nltk.downloader -d /usr/share/nltk_data/ punkt
 
 # clone checks
 RUN git clone -b master https://github.com/cs50/checks.git /home/ubuntu/.local/share/check50/cs50/checks
 
 # check50 wrapper
-COPY ./check50-wrapper /usr/local/bin/ 
+COPY ./check50-wrapper /usr/local/bin/
```

Comment:
This shouldn't be in a common layer, I assume?

Reply:
It's not.